mayastor/src/grpc/mayastor_grpc.rs

```diff
@@ -123,6 +124,23 @@ impl MayastorSvc {
     }
 }
 
+impl TryFrom<CreatePoolRequest> for PoolArgs {
+    type Error = LvsError;
+    fn try_from(args: CreatePoolRequest) -> Result<Self, Self::Error> {
+        match args.disks.len() {
+            0 => Err(LvsError::Invalid {
+                source: Errno::EINVAL,
+                msg: "invalid argument, missing devices".to_string(),
+            }),
+            _ => Ok(Self {
+                name: args.name,
+                disks: args.disks,
+                uuid: None,
```

Comment:
Shouldn't the uuid be filled from `args` ?

Reply:
This is from the old service so did not want to change the existing behavior. This will only be taken for mayastor.v1 services.

Reply:
Ah right, makes sense!